Assets/Scripts/Layers/Adapters/CSVImportAdapter.cs

```diff
@@ -50,13 +51,15 @@ static Transform DatasetLayerParent
         }
 
         public int maxParsesPerFrame = 100;
+        [SerializeField] private StringEvent csvOverwriteMessageEvent;
```

Comment:
Ik denk dat het wijs is om ipv. ons EventSystem te gebruiken als dependency hier UnityEvent< string > te gebruiken. Je kan dan alsnog die SO's gebruiken, maar hebben we geen dependencies in de scripts meer zitten